components/eventtype/CreateEventType.tsx

```diff
@@ -81,18 +81,21 @@ export default function CreateEventTypeButton(props: Props) {
 
   // inject selection data into url for correct router history
   const openModal = (option: EventTypeParent) => {
-    router.push({
-      pathname: router.pathname,
-      query: {
-        ...router.query,
-        new: "1",
-        eventPage: option.slug,
-        ...(option.teamId
-          ? {
-              teamId: option.teamId,
-            }
-          : {}),
-      },
+    // setTimeout fixes a bug where the url query params are removed immediately after opening the modal
+    setTimeout(() => {
```

Comment:
It's not the best fix I know, but it's a fast fix since this bug is now in production. Despite 20 minutes of debugging I couldn't find the source of why the URL params are stripped the second time the modal is opened. It looks like a race condition with react router, even though I removed a seemingly pointless `router.push` call from the original code, there isn't anything else in this component calling functions on the router.